skbio/sequence/_dna.py

```diff
@@ -222,6 +220,7 @@ def translate(self, *args, **kwargs):
 
         See Also
         --------
+        RNA.reverse_transcribe
```

Comment:
Can you remove this line and `transcribe`? I think I accidentally added `transcribe` when writing this code. The see also block should just point to translation methods.